src/services/AuthServices.js

```diff
@@ -19,10 +19,21 @@ export const getCurrentUser = () => {
 };
 export const removeCurrentUser = () => LocalStorageService.removeSessionToken();
 
-export const login = () =>
-  // TODO: Implement call to authentication API here
-  new Promise(resolve => {
-    setTimeout(() => {
-      resolve({ ok: true, data: { sessionToken: 'token' } });
-    }, 750); // eslint-disable-line no-magic-numbers
+const getTokenFromResponse = queryParams =>
+  queryParams.match(/access_token=[a-z0-9]+/g)[0].substring('access_token='.length);
+
+export const storeToken = response => {
+  const token = getTokenFromResponse(response.data);
+  localStorage.setItem('auth_token', token);
+};
```

Comment:
I think to store the token you need the `LocalStorageService`, that way it will be encoded and not so accessible by other people 😄